templates/profile/prometheus/exporter/mysql/target.yaml.erb

```diff
@@ -1,5 +1,5 @@
-- targets: [ '<%= @ipaddress %>:9101' ]
+- targets: [ '<%= @ipaddress %>:9100' ]
   labels:
     datacenter: '<%= @datacenter %>'
     hostname: '<%= @hostname %>'
-    priority: '<%= @master ? 'primary' : 'backup' %>'
+    role: '<%= @role %>'
```

Comment:
I'm confused by this change. While `priority` is certainly not the right word, it does seem like it'd be useful to track whether a database is a master or slave (although, maybe even that is superfluous given our master-master architecture). In any case, I'm not sure there's much value in tracking a server's role, in the context of mysql, but I haven't given it much thought.

Reply:
I wasn't sure what to do in this spot.  I figured the priority was wrong, since all mysql servers are technically writable.   I defaulted to "role" without really understanding what it means.    Perhaps, nothing is necessary here?

Reply:
Hm yeah, maybe just delete the line. The main reason we're doing this is to detect replication errors that require our attention, and there isn't really an easy way to rate different servers—even if we're talking about some bronze-level service, replication errors are bad news in danger of getting worse and worse, so we want to resolve them anytime they happen.

With haproxy, there was a good reason to know whether something was primary/backup, since backup problems are only bad if the primary is also bad.

Actually, I've come full circle, and I back your decision to add the role (although I bet right now, as written, this template will always produce `role: ''`). I could easily imagine wanting to say, collect database metrics for www-lib, and that should hinge on a role, rather than e.g. needing to change every time we replace a database server.